analyses/tp53_nf1_score/02-qc-rna_expression_score.Rmd

```diff
@@ -45,45 +45,59 @@ score_stranded_df <- read_tsv(file.path(results_dir, "pbta-gene-expression-rsem-
 
 ```
 
- 2) Read in scores from tp53-nf1-classifier for polya
-```{r}
-score_polya_df <- read_tsv(file.path(results_dir, "pbta-gene-expression-rsem-fpkm-collapsed.polya_classifier_scores.tsv")) %>%
-  dplyr::select(
-    "tp53_score",
-    "sample_id"
-  ) %>%
-  as.data.frame()
-
-```
 
 ### Expression matrices
 
+Read in expression matrices and filter expression matrices to only contain cohort and samples of interest using histology file
+
 ```{r}
 
-stranded_exp_file <- file.path(data_dir,"pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds")
-polya_exp_file <- file.path(data_dir,"pbta-gene-expression-rsem-fpkm-collapsed.polya.rds")
+exp_file <- file.path(data_dir,"gene-expression-rsem-tpm-collapsed.rds")
+expressionMatrix<-readRDS(exp_file) 
+
+clinical_filtered <- clinical %>% dplyr::filter(sample_type == "Tumor") %>%
+  dplyr::filter(experimental_strategy == "RNA-Seq") %>% 
+  dplyr::filter(cohort %in% c("PBTA", "GMKF", "TARGET")) 
 
+# filter to samples of a particular RNA library type
+polya_samples <- clinical_filtered %>% dplyr::filter(RNA_library == "poly-A") %>%
+  dplyr::pull(Kids_First_Biospecimen_ID)
+stranded_samples <- clinical_filtered %>% dplyr::filter(RNA_library == "stranded") %>%
+  dplyr::pull(Kids_First_Biospecimen_ID)
+polya_stranded_samples <- clinical_filtered %>% dplyr::filter(RNA_library == "poly-A stranded") %>%
+  dplyr::pull(Kids_First_Biospecimen_ID)
```

Comment:
I think we may want to generalize this to perhaps check all values (we will have new coming in with v10), and create the types based on the levels of `RNA_library`... not sure yet

Reply:
@jharenza, good point! What I can do is to get the unique `RNA_library` and then iterate through the list? That way, it would be more generalizable.

Reply:
yeah, I think that would be good